multirange.js

```diff
@@ -59,8 +59,38 @@ self.multirange = function(input) {
 		ghost.style.setProperty("--high", input.valueHigh * 100 / max - 1 + "%");
 	}
 
-	input.addEventListener("input", update);
-	ghost.addEventListener("input", update);
+    function passClick(evt) {
+      // Are the ghost and input elements inverted? (ghost is lower range)
+      var isInverted = input.valueLow == ghost.value;
+      // Find the horizontal position that was clicked (as a percentage of the element's width)
+      var clickPoint = evt.offsetX / this.offsetWidth;
+      // Map the percentage to a value in the range (note, assumes a min value of 0)
+      var clickValue = max * clickPoint;
+
+      // Get the distance to both high and low values in the range
+      var highDiff = Math.abs(input.valueHigh - clickValue);
+      var lowDiff = Math.abs(input.valueLow - clickValue);
+
+      if (lowDiff < highDiff && !isInverted || (isInverted && lowDiff > highDiff)) {
```

Comment:
I would change to:
```js
if (isInverted? highDiff < lowDiff : lowDiff < highDiff)
```

There's probably an even more elegant way to do this, but it's escaping me now.

Reply:
(lowDiff > highDiff) == isInverted